src/index.js

```diff
@@ -263,7 +263,7 @@ class Anvil {
       throw new Error(`dataType must be one of: ${supportedDataTypes.join('|')}`)
     }
 
-    const versionNumber = clientOptions?.versionNumber
+    const versionNumber = clientOptions.versionNumber
```

Comment:
Cool, looks like we access clientOptions above so no need for the ?

Reply:
Yeah, was just there in case someone puts in an explicit `undefined` / `null`, but I guess at that point they shouldn't do that.